managed/ui/src/components/common/associatedUniverse/AssociatedUniverse.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { YBModal } from '../forms/fields';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import { Link } from 'react-router';
+import './AssociatedUniverse.scss';
+
+function AssociatedUniverse(props) {
+  const { onHide, associatedUniverses, visible, title } = props;
+  /**
+   * Returns the decorated status.
+   * @param universeStatus status of universe.
+   */
+  const modifyStatus = (item,row) => {
+    const universeStatus = row.universePaused
+      ? 'Paused'
+      : row.updateSucceeded && !row.updateInProgress
```

Comment:
For now, universe status is based on the task progress. Ask BE guys to add a standalone universe status field that will tell us the current universe status. Make a TODO here for that.

Reply:
Done